package NAntiRobot;

message TEvlogRow {
    optional bytes event = 1;
    optional string source_uri = 2;
}
